eks_fargate/README.md

```diff
@@ -144,6 +152,47 @@ spec:
 
 **Note**: Add your desired `kube_cluster_name:<CLUSTER_NAME>` to the list of `DD_TAGS` to ensure your metrics are tagged by your desired cluster. You can append additional tags here as space separated `<KEY>:<VALUE>` tags.
 
+#### Running the Cluster Agent / Cluster Checks Runner
+
+Running the Cluster Agent is recommended as it is required for a number of features: events collection, Kubernetes resources view, cluster checks, etc.
+When using EKS Fargate, two scenarios are possible depending if the EKS cluster is runnning mixed workloads (Fargate/non-Fargate) or not.
+
+If the EKS cluster runs Fargate and non-Fargate workloads **AND** you want to monitor non-Fargate workload through Node Agent DaemonSet, the Cluster Agent/Cluster Checks Runner can be easily be added to this deployment.
+Follow the [Cluster Agent Setup][25] documentation to deploy it.
+
+One specifity is that the Cluster Agent token must be reachable from the Fargate tasks you want to monitor, which is not the case by default using our Helm Chart or Datadog Operator as it creates a secret in the target namespace.
```

Comment:
```suggestion
The Cluster Agent token must be reachable from the Fargate tasks you want to monitor. If you are using the Helm Chart or Datadog Operator, this is not reachable by default because a secret in the target namespace is created.
```